app/models/support/service_performance.rb

```diff
@@ -33,40 +33,56 @@ def number_of_centrally_managed_schools
       .count
   end
 
-  def total_devices_available
+  def total_devices_allocated
+    total_devices_cap - total_devices_over_ordered
+  end
+
+  def total_devices_cap
     sum_allocation(Arel.sql("CASE order_state WHEN 'cannot_order'
                                         THEN raw_laptops_ordered
                                         ELSE raw_laptop_allocation + over_order_reclaimed_laptops + circumstances_laptops
                                     END"))
   end
```

Comment:
Maybe we should memoise this as it it is called twice?

Reply:
Sure, all of these totals should be memoized